server/initializers/config.ts

```diff
@@ -83,7 +83,8 @@ const CONFIG = {
   },
   TRUST_PROXY: config.get<string[]>('trust_proxy'),
   LOG: {
-    LEVEL: config.get<string>('log.level')
+    LEVEL: config.get<string>('log.level'),
+    ROTATION: config.get<boolean>('log.rotation')
```

Comment:
`log.rotation.enabled`

Reply:
Indeed, I completely forgot this modification, thank you for your look-out. I apologize.